_data-prepper/pipelines/configuration/processors/write_json.md

```diff
@@ -13,6 +13,6 @@ The `write_json` processor converts an object in an event into a JSON string. Yo
 
 | Option | Description | Example |
 | :--- | :--- | :--- |
-| source | Mandatory field that specifies the name of the field in the event containing the message or object to be parsed. | If `source` is set to `"message"` and the input is `{"message": {"key1":"value1", "key2":{"key3":"value3"}}`, then the `write_json` processor generates `{"message": "{\"key1\":\"value`\", \"key2\":"{\"key3\":\"value3\"}"}"`.
-| target | An optional field that specifies the name of the field in which the resulting JSON string should be stored. If `target` is not specified, then the `source` field is used.
+| source | Mandatory field that specifies the name of the field in the event containing the message or object to be parsed. | If `source` is set to `"message"` and the input is `{"message": {"key1":"value1", "key2":{"key3":"value3"}}}`, then the `write_json` processor generates `{"message": "{\"key1\":\"value\", \"key2\":{\"key3\":\"value3\"}}"}`.
```

Comment:
Is there a reason the example needed to change? Also, adding the line at the end will fix the table.

```suggestion
| source | Mandatory field that specifies the name of the field in the event containing the message or object to be parsed. | If `source` is set to `"message"` and the input is `{"message": {"key1":"value1", "key2":{"key3":"value3"}}}`, then the `write_json` processor generates `{"message": "{\"key1\":\"value\", \"key2\":{\"key3\":\"value3\"}"}"`. |
```

Reply:
I was trying to fix the formatting:
<img width="879" alt="image" src="https://github.com/opensearch-project/documentation-website/assets/104463754/c3c12df1-9aec-4b2a-a070-a7edcdd06cf3">

Reply:
@dlvenable is going to take a look and help clarify.